zulip/zulip/__init__.py

```diff
@@ -1539,6 +1539,23 @@ def get_storage(self, request: Optional[Dict[str, Any]] = None) -> Dict[str, Any
             request=request,
         )
 
+    def remove_storage(self, request: Optional[Dict[str, Any]] = None) -> Dict[str, Any]:
+        '''
+            Example usage:
+
+            >>> client.update_storage({'storage': {"entry 1": "value 1", "entry 2": "value 2", "entry 3": "value 3"}})
+            >>> client.get_storage()
+            {'result': 'success',  'msg': '', 'storage': {"entry 1": "value 1", "entry 2": "value 2", "entry 3": "value 3"}}
+            >>> client.remove_storage({'keys': ["entry 1", "entry 3"]})
+            >>> client.get_storage()
+            {'result': 'success',  'msg': '', 'storage': {'entry 2': 'value 2'}}
+        '''
+        return self.call_endpoint(
+            url='bot_storage',
+            method='DELETE',
+            request=request,
+        )
```

Comment:
Can you change the `request` format to use the kwargs format for passing parameters (see e.g. #579 )?

Otherwise, this lgtm.

Reply:
@timabbott I have changed `request` to use kwargs format and changed the rest of the pr accordingly.
Hope it meets your description.